gems/ibm_cloud/ibm_cloud.gemspec

```diff
@@ -0,0 +1,31 @@
+require_relative 'lib/ibm_cloud/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "ibm_cloud"
+  spec.version       = IbmCloud::VERSION
+  spec.authors       = ["Adam Grare"]
+  spec.email         = ["agrare@redhat.com"]
```

Comment:
Should we make this more general?

Reply:
:point_up: **Very WIP** 

:laughing: Literally just generated this with default settings to start playing with it

Reply:
Oh yeah, I expected as much when I saw **Very WIP** - figured I'd comment anyway to TODO list it.